github3/pulls.py

```diff
@@ -88,11 +91,8 @@ def contents(self):
         :param stream: When true, the resulting object can be iterated over via
             ``iter_contents``.
         """
-        headers = {'Accept': 'application/octet-stream'}
-        resp = self._get(self.raw_url, headers=headers)
-        if self._boolean(resp, 200, 404):
-            return resp.content
-        return b''
+        json = self._json(self._get(self.contents_url), 200)
+        return self._instance_or_null(Contents, json)
 
     def download(self, path=None):
```

Comment:
I'd be okay with just deleting the download method. This is the v1.0 branch so I'm not particularly concerned with backwards compat. v1.0 hasn't been released yet. Just add a note to the breaking changes section in https://github.com/sigmavirus24/github3.py/blob/develop/LATEST_VERSION_NOTES.rst